index.html

```diff
@@ -76,6 +76,13 @@
 	$generatedURL = $('#generated-url');
 $form.submit(function(e){
 	e.preventDefault();
+	window.open($generatedURL.attr('href'), '_blank');

```

Comment:
Hmm, do you test in other browsers if the new window would be blocked when 'enter'-ing the form?